src/components/registration/steps.jsx

```diff
@@ -27,6 +27,17 @@ var Tooltip = require('../../components/tooltip/tooltip.jsx');
 require('./steps.scss');
 
 var DEFAULT_COUNTRY = 'us';
+var COUNTRY_OPTIONS = countryData.countryOptions.concat({
+    label: <intl.FormattedMessage id="teacherRegistration.selectCountry" />,
+    disabled: true,
+    selected: true
+}).sort(function (a, b) {
+    if (a.disabled) return -1;
+    if (b.disabled) return 1;
+    if (a.value === DEFAULT_COUNTRY) return -1;
```

Comment:
This makes US always be at the top, but the previous behavior would put the user's last country selection at the top. If it were a function that took a default_country argument, you could pass it `this.props.default_country` and it would behave exactly as before.